src/snowcli/config.py

```diff
@@ -39,63 +84,40 @@ def _add_options(self):
             parse_str=tomlkit.parse,
         )
 
-    def get_section(self, *path) -> TOMLDocument:
+    def _find_section(self, *path) -> TOMLDocument:
         section = self
         idx = 0
         while idx < len(path):
             section = section[path[idx]]
             idx += 1
         return section
 
-    def section_exists(self, *path) -> bool:
-        try:
-            self.get_section(*path)
-            return True
-        except NonExistentKey:
-            return False
+    def _merge_section_with_env(self, section: Table, *path) -> Dict[str, str]:
+        merged_dict = {}
+        for k in section:
```

Comment:
Does it mean all env variables have to be also specified in toml? For example if I want to set password in env variable, then I have to provide some dummy password in toml?

Reply:
Yes

Reply:
Don't we remove some part of functionality which we had before (or thought we had)?: https://github.com/Snowflake-Labs/snowcli#use-environment-variables-for-snowflake-credentials

Reply:
I returned to previous solution